hack/run-locally.sh

```diff
@@ -94,11 +101,64 @@ function wait_for_cluster() {
     fi
 }
 
+function from_scratch() {
+    if [[ -n "${INSTALLER_DIR:-}" ]]; then
+        return 0
+    fi
+    return 1
+}
+
+PLUGIN_IMAGE=${PLUGIN_IMAGE:-}
```

Comment:
Always put quotes around variable expansions. (And for `${FOO}/bar/baz`, put quotes around the whole path.)